static/css/global.css

```
@@ -27,6 +27,9 @@
   margin-left: -1.3em;
   width: 1em;
 }
+.button.is-info {
+  background-color: #236BE4;
```

@MeliAlbu acá te volvió a faltar usar el color como variable
```suggestion
  background-color: var(--blue);
```